doc/client.rst

```diff
@@ -71,7 +71,7 @@ Connection
             Python client 5.0.0 and up will fail to connect to Aerospike server 4.8.x or older.
             If you see the error "-10, ‘Failed to connect’", please make sure you are using server 4.9 or later.
 
-        .. seealso:: `Security features article <https://docs.aerospike.com/server/guide/security/index.html>`_.
+        .. seealso:: `Security features article <https://aerospike.com/docs/server/guide/security/index.html>`_.
```

Comment:
See comment below

Reply:
Should be https://aerospike.com/docs/server/guide/security